src/replay/index.ts

```diff
@@ -144,11 +147,13 @@ export class Replayer {
     if (!this.config.logConfig.replayLogger)
       this.config.logConfig.replayLogger = this.getConsoleLogger();
 
+    this.diggestAutoFit = this.diggestAutoFit.bind(this)
     this.handleResize = this.handleResize.bind(this);
     this.getCastFn = this.getCastFn.bind(this);
     this.emitter.on(ReplayerEvents.Resize, this.handleResize as Handler);
 
     this.setupDom();
+    this.setUpAutoFit.bind(this)()
```

Comment:
No need to bind them as they aren't used as callback anywhere
```suggestion
    this.handleResize = this.handleResize.bind(this);
    this.getCastFn = this.getCastFn.bind(this);
    this.emitter.on(ReplayerEvents.Resize, this.handleResize as Handler);

    this.setupDom();
    this.setUpAutoFit()
```